_authors/dmitrypol.markdown

```diff
@@ -0,0 +1,12 @@
+---
+name: Dmitry Polyakovsky
+short_name: dmitrypol
+photo: '/assets/media/authors/dmitrypol.jpeg'
+github: dmitrypol
+---
+
+Dmitry is the Consulting Member of Technical Staff at Oracle Cloud.  
```

Comment:
```suggestion
Dmitry is a Consulting Member of Technical Staff at Oracle Cloud.  
```

Unless you are the only one?

Reply:
thank you

Reply:
I thought you were the singular important one ;)